modules/distr-tracing-otel-forwarding.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * /distr_tracing/distr_tracing_otel/distr-tracing-otel-using.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-forwarding_{context}"]
+= Forwarding traces to a TempoStack by using the OpenTelemetry Collector
+
+To configure log forwarding to a TempoStack, you can deploy and configure the OpenTelemetry Collector. You can deploy the OpenTelemetry Collector in the deployment mode by using the specified processors, receivers, and exporters. For other modes, see the OpenTelemetry Collector documentation linked in _Additional resources_.
+
+.Prerequisites
+
+* The {OTELOperator} is installed.
+* The {TempoOperator} is installed.
+* A TempoStack is deployed on the cluster.
+
+.Procedure
+
+. Create a service account for the OpenTelemetry Collector.
++
+.ServiceAccount
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+----
+====
+
+. Create a cluster role for the service account.
++
+.ClusterRole
+====
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+  <1>
+  <2>
```

Comment:
Is there an example value you could include here?

Reply:
Can we include in a subsequent PR? (To give you the context, this is a PR for the docs release of a product version that was already released last week, so I'd like to avoid further delays due to details that can be improved in another PR if possible.)

Reply:
Yes, totally agree with that decision :+1:

Reply:
Thank you so much for understanding.